ci/ray_ci/container.py

```diff
@@ -32,12 +33,42 @@ def __init__(self, docker_tag: str, volumes: Optional[List[str]] = None) -> None
         self.docker_tag = docker_tag
         self.volumes = volumes or []
 
-    def run_script(self, script: List[str]) -> bytes:
+    def run_script_and_output(self, script: List[str]) -> bytes:
         """
-        Run a script in container
+        Run a script in container and returns its output
         """
         return subprocess.check_output(self._get_run_command(script))
 
+    def run_script(self, script: List[str]) -> None:
+        """
+        Run a script in container
+        """
+        return subprocess.check_call(
+            self._get_run_command(script),
+            stdout=sys.stdout,
+            stderr=sys.stderr,
+        )
+
+    def install_ray(self) -> None:
```

Comment:
this is the same as setup_test_environment in [1], just move to the parent class